packages/react-components/react-button/library/src/components/SplitButton/useSplitButtonStyles.styles.ts

```diff
@@ -171,18 +172,40 @@ const useRootStyles = makeStyles({
   },
 });
 
+const useIconOnlyStyles = makeStyles({
+  outline: {},
+  secondary: {},
+  primary: {},
+  subtle: {},
+  transparent: {
+    [`& .${splitButtonClassNames.primaryActionButton}`]: {
+      paddingRight: tokens.spacingVerticalNone,
+    },
+    [`& .${splitButtonClassNames.menuButton}`]: {
+      paddingLeft: tokens.spacingVerticalNone,
+      paddingRight: tokens.spacingVerticalNone,
+    },
+  },
+});
+
 export const useSplitButtonStyles_unstable = (state: SplitButtonState): SplitButtonState => {
   'use no memo';
 
   const rootStyles = useRootStyles();
   const focusStyles = useFocusStyles();
+  const iconOnlyStyles = useIconOnlyStyles();
 
   const { appearance, disabled, disabledFocusable } = state;
 
+  const noIconClass = state.primaryActionButton
```

Comment:
_nit:_ This is really icon only styles, so I would rename it to something like `iconOnlyClassName`

```suggestion
  const iconOnlyClassName = state.primaryActionButton
```